package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "build": "webpack",
+    "build": "NODE_ENV=production webpack",
```

Comment:
I think it would be better to set this in circle than to enforce it here. Your `.env` file is loaded before any webpack stuff is run so you can use it to set the variable for local builds.